### DISCUSSION TITLE
Markdown page renders blank

### DISCUSSION THREAD

**Discussion by djwoodhouse** [requarks/wiki#6533] [Error / Bug Report]
The following markdown when published produces a blank page
# Lenz's Law

Inductance in its various forms is strongly linked to the electromotive force (emf) produced by the rate of change of magnetic flux within an area, as described by Lenz's Law.
$$\epsilon = -{{d\phi}\over{dt}}$$
where $\phi = \bar{B}.\bar{A}$

In this section we look at the various physical mechanisms associated with the various forms of Lenz's Law.

## Scalar Derivative
This is the usual form which Lenz's Law is used to describe in devices such as solenoids and transformers. In this form the flux, a scalar, is calculated as the product of the two vectors  $\phi = \bar{B}.\bar{A}$

Add picture of field thru coil ...

## Area Derivative
By using the mathemtical law of the derivative of the product of functions Lenz's Law may be expanded as follows: 

## Field Derivative
Say something

## Conclusion
So there are three different physical mechanisms which produce emf. These mechanisms don't have to occur independently.

**Comment by djwoodhouse** [comment]
Further to above, the page renders if all the equations are removed!

**Reply by NGPixel** [reply]
Check the server process logs, the page probably fails during rendering. Are you using the latest version?

**Comment by djwoodhouse** [comment]
Hi Nicolas,

The sever logs don't show anything abnormal.

Yes I am using the latest version.

Regards Darren
